.github/workflows/ci.yaml

```diff
@@ -0,0 +1,122 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request: {}
+  workflow_dispatch: {}
+
+env:
+  # Common versions
+  GO_VERSION: '1.16'
+  GOLANGCI_VERSION: 'v1.31'
+  DOCKER_BUILDX_VERSION: 'v0.4.2'
+
+jobs:
+
+  check-license:
+    if: github.repository == 'openyurtio/yurt-app-manager'
```

Comment:
Why add the if check?

Reply:
only check openyurtio/yurt-app-manager repo, skip repos like rambohe-ch/yurt-app-manager

Reply:
I think it's better for contributors to trigger the github actions on their own forked repo to debug first before submitting a pr to the openyurt repo.

Reply:
> I think it's better for contributors to trigger the github actions on their own forked repo to debug first before submitting a pr to the openyurt repo.

Yes, I agree.

Reply:
fixed.